src/Utilities/ToolTask.cs

```diff
@@ -659,12 +659,12 @@ private string GetTemporaryResponseFile(string responseFileCommands, out string
         }
 
         /// <summary>
-        /// Expect task to override this method if they need information about the process or process events during task execution.
+        /// We expect tasks to override this method if they need information about the tool process or its process events during task execution.
         /// Implementation should make sure that the task is started in this method.
```

Comment:
nit: `task` -> `process`? Or maybe remove this line as this is included in the `<param>` and `<returns>` sections below?

Reply:
I went with `task` as it is what is written in the other comments for function in the `ToolTask`. I think we can leave it as it described the intention of the method, and it also adds that this function will be called during the execution.

Reply:
I see, makes sense to leave it then. Thank you!